TripPlanner - Project/TripPlannerFrontend/src/pages/Home.tsx

```diff
@@ -20,7 +21,8 @@ export const Home = () => {
   }, 1000);
 
   return (
-    <DefaultLayout gapRow={3} marginVertical={5} marginHorizontal="auto">
+    <DefaultLayout gapRow={3} marginHorizontal="auto">
+      <Header />
```

Comment:
Since header will always be present, I think importing it in `src/index.tsx` will be better. We want to display header everywhere not only in `/` (home route)